README.md

```diff
@@ -34,6 +33,121 @@ Special notes:
  * Since we cannot distribute Java Richards, a patch is required to derive the
    tracing version (`patches/trace_java_richards.diff`)
 
+## Using the scripts here to analyse an existing benchmark
+
+### Mandatory requirements
+
+  * Python 2.7 - the code here is not Python 3.x ready
+  * bzip2 / bunzip2 and bzip2 headers
+  * curl and headers
```

Comment:
Can we put something like Curl (including its header files)? "and headers" is a little vague, and not obviously relevant on OSs that routinely package headers with programmes.